connector/dummy.go

```diff
@@ -150,10 +108,16 @@ func (conn *Dummy) CreateLabel(ctx context.Context, name []string) (imap.Mailbox
 	return mbox, nil
 }
 
-func (conn *Dummy) UpdateLabel(ctx context.Context, labelID imap.LabelID, name []string) error {
-	conn.state.updateLabel(labelID, name)
+func (conn *Dummy) UpdateLabel(ctx context.Context, labelID imap.LabelID, newName []string) error {
+	oldName := conn.state.getLabel(labelID).Name
```

Comment:
missing error handling for invalid label id?

Reply:
eh it's just the dummy -- if we're passing in missing label IDs it's probably worth just panicking cos something's seriously wrong

Reply:
But then you can't verify the rest of your code handles this gracefully.

Reply:
Fair enough, added some basic error handling.